tests/src/test/java/alluxio/client/fs/FileSystemMasterRestartIntegrationTest.java

```diff
@@ -363,6 +364,7 @@ public void updateAccessTimeAsyncFlush() throws Exception {
    * Tests journal is not updated with access time asynchronously after delete.
    */
   @Test
+  @LocalAlluxioClusterResource.Config(confParams = {PropertyKey.Name.MASTER_METASTORE, "HEAP"})
```

Comment:
The 2 tests in this file is rough to translate to ROCKS because it has to file masters up which runs into lock conflict in ROCKS.